frontend/components/site/SiteSettings/ReportConfigs.jsx

```diff
@@ -108,7 +108,17 @@ function ReportConfigs({ siteId: id }) {
 
   function ruleRender(rule) {
     const isPagesRule = rule?.source === 'Pages';
-    const placeholder = (rule.match?.join(', ') || '') + (isPagesRule ? ' (not editable - suppressed by Pages)' : '/assets class="ignore"');
+    const customPlaceholder = '/assets class="ignore"';
+    function pagesPlaceholder() {
+      if (rule.match) {
+        return rule.match.map(match => (
```

Comment:
Will `rule.match` always exist as an array? Should we add a check in the `if`?

Reply:
@drewbo I'm not actually sure about this one

Reply:
yes, it's a little fragile but it's always set to an array via https://github.com/cloud-gov/pages-core/blob/main/frontend/components/site/SiteSettings/ReportConfigs.jsx#L64